fabtests/common/shared.c

```diff
@@ -108,6 +108,7 @@ struct timespec start, end;
 int listen_sock = -1;
 int sock = -1;
 int oob_sock = -1;
+bool use_rx_cq_data = true;
```

Comment:
Would `allow_rx_cq_data` more suitable since rx cq data is only really used when the provider requires it.

Reply:
Updated